packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -4511,6 +4511,111 @@ describeWithDOM('mount', () => {
       });
     });
 
+    describeIf(is('>= 16'), 'componentDidCatch', () => {
+      describe('errors inside an error boundary', () => {
+        const errorToThrow = new EvalError('threw an error!');
+        // in React 16.0 - 16.2, and sole older nodes, the actual error thrown isn't reported.
```

Comment:
I don't fully understand this comment: `in React 16.0 - 16.2, and sole older nodes`.

Reply:
oops, typo for “some”